lib/ui/widget/dimmed_image_background.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+
+import 'blurred_background.dart';
+import 'dimmed_background.dart';
+
+class DimmedImageBackground extends StatefulWidget {
+  final ImageProvider image;
+  final bool shouldBlur;
+  final Animation<double>? backgroundAnimation;
+
+  const DimmedImageBackground({
+    super.key,
+    required this.image,
+    this.shouldBlur = false,
+    this.backgroundAnimation,
```

Comment:
What do think about letting the widget handle the animation itself and give a flag `animation` to do and don't animate? 🤔

Reply:
@markgravity updated in https://github.com/nimblehq/ic-flutter-avishek/pull/32/commits/e5ec4ea536253835b8f95586006b77e58e75451b.